src/content/experimental/layout/layout.md

```diff
@@ -56,11 +56,11 @@ A [fluid grid](#fluid-grid) divides space by two. This division is immediately e
 | 8 |
 | 16 |
 
-For dense content, use mini units for further alignment and spatial guidance.
+For dense content, use Mini Units for further alignment and spatial guidance.
 
 ![grid customization](images/grid_customization.png)
 
-A [fixed grid](#fixed-grid) starts with a fixed unit size from the [sizing scale](#sizing-scale), then tiles and wraps as needed, like text. To employ the 2x concept, multiply box sizes by two and mix them together.
+A [fixed grid](#fixed-boxes) starts with a fixed unit size from the [sizing scale](#sizing-scale), then tiles and wraps as needed, like text. To employ the 2x concept, choose a base unit from the sizing scale increase your container size using multiples of two.
```

Comment:
This needs grammar work:
> choose a base unit from the sizing scale increase your container size using multiples of two

Also, the previous sentence mentioned the sizing scale. Perhaps just modify the previous sentence to say "choose" instead of "starts with"?

Reply:
Ahhh, I'm confusing the idea of the box or container here with idea of the initial unit or increment from the sizing scale. When you said "multiply box sizes by two and mix them together" I thought you were talking about multiplying your increment by multiples of two to drive your container size.

Now that I understand... we could go back to your original... or include something a little more instructive about how to size your container. 

"With a fixed grid, your container size and aspect ratio is fixed; containers will tile and wrap as needed, like text. To size your container, choose a fixed unit size from the sizing scale. Map multiples of your chosen fixed unit to a recommended aspect ratio (anchor link). To employ the 2x concept to a fixed grid , multiply your container sizes by two, then mix and match."